source/DataWriter.h

```diff
@@ -34,6 +34,7 @@ class DataNode;
 class DataWriter {
 public:
 	// Constructor, specifying the file to write.
+	// An empty path will create a DataWriter that will not save its contents.
```

Comment:
I would prefer another constructor that just takes no path

Reply:
Done